data/simulations/FAH_SARS-CoV-2_spike.yml

```diff
@@ -0,0 +1,32 @@
+type: md
+title: Trajectories of the SARS-CoV-2 spike protein from Folding@Home
+description: All-atom MD simulations of the SARS-CoV-2 spike protein, simulated using Folding@Home. Simulations were run using either GROMACS or OPENMM and are stored as compressed binary XTC files. The dataset comprises 3 projects, each having a RUN*/CLONE*/result* directory structure. Each RUN represents a unique starting conformation, each CLONE is a unique MD run from the specified starting conformation, and each result is a fragment of the contiguous simulation.
+creator: Folding@Home
+institute: Washington University in St. Louis
+proteins:
+    - spike
+structures:
+    - 6VXX
+models:
+    - FAH_SARS-CoV-2_spike
+files:
+    - s3://fah-public-data-covid-19-cryptic-pockets/SARS-CoV-2/spike/PROJ14217/
```

Comment:
This is meant for links to input files

Reply:
If I understand properly, you're saying that the S3 bucket locations go in the description and `files` be a list of the inputs and models used for simulations?